packages/cms-data-sync/test/run-migrations.test.ts

```diff
@@ -131,6 +132,16 @@ jest.mock('../src/discover/discover.data-migration', () => {
   };
 });
 
+var mockMigrateResearchTags: jest.MockedFunction<typeof migrateResearchTags>;
```

Comment:
have you tried using const for this? i tried in here https://github.com/yldio/asap-hub/blob/25f049fcd703e11c0ec19023ddef05d8d604b4d4/packages/server-common/test/handlers/webhooks/contentful.handler.test.ts#L17 and it worked

Reply:
See above re. wholesale copy-pasting. I'll probably change them all because there's no good reason to use `var` that I'm aware of.

Reply:
nostalgia

Reply:
So apparently there is a good reason. If I either swap out the `var` directly for a `let`, or `const` with the assignment to `jest.fn()` moved inline then I get the following error.

```
ReferenceError: Cannot access 'mockMigrateEvents' before initialization
```

I think it works in the example you linked because the mock is the return value of an imported function rather than the import itself.